python/ray/autoscaler/_private/commands.py

```diff
@@ -137,16 +137,18 @@ def request_resources(num_cpus: Optional[int] = None,
         overwrite=True)
 
 
-def create_or_update_cluster(config_file: str,
-                             override_min_workers: Optional[int],
-                             override_max_workers: Optional[int],
-                             no_restart: bool,
-                             restart_only: bool,
-                             yes: bool,
-                             override_cluster_name: Optional[str] = None,
-                             no_config_cache: bool = False,
-                             redirect_command_output: Optional[bool] = False,
-                             use_login_shells: bool = True) -> Dict[str, Any]:
+def create_or_update_cluster(
```

Comment:
do you need to also update the sdk `create_or_update_cluster` ?

Reply:
hmm, sure I could, at least for the sake of consistency add `no_monitor_on_head: bool = False` to the `commands.py` function call inside of `sdk.create_or_update_cluster`

Probably don't want to add this yet to the arguments of `sdk.create_or_update_cluster`

The operator currently uses the `commands.create_or_update_cluster`